libs/core/utils/types.ts

```diff
@@ -0,0 +1,44 @@
+/** Type. */
+export enum Type {
+	'TV',
+	'OVA',
+	'MOVIE',
+	'SPECIAL',
+	'ONA',
+	'MUSIC',
+	'UNKNOWN',
```

Comment:
let's improve this:
```suggestion
	TV = 'TV',
	OVA = 'OVA',
	Movie = 'MOVIE',
	Special = 'SPECIAL',
	ONA = 'ONA',
	Music = 'MUSIC',
	Unknown = 'UNKNOWN',
```

'Why?' you can ask me: 
- The fact is that even if the enum values somehow change, we will not have to change anything in the application.
- Regardless of the values, we will be able to control the naming in our application.